M2/LICENSING-NOTES

```diff
@@ -49,4 +48,4 @@ modifications by Paul Zinn-Justin			public domain
 
 for M2 binaries:
 
-We are prevented from using GPL 2 by mpir and mpfr
+We are prevented from using GPL 2 by mpfr
```

Comment:
Huh, [mpfr.org](https://www.mpfr.org/) says:
> MPFR is free. It is distributed under the [GNU Lesser General Public License](https://www.gnu.org/licenses/lgpl-3.0.html) (GNU Lesser GPL), version 3 or later (2.1 or later for MPFR versions until 2.4.x).

I didn't know this file existed.

Reply:
Apparently GPL-2 and LGPL-3 aren't compatible: https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility

Reply:
Ah, I see.